Tests/XCUITests/PerformanceTests.swift

```diff
@@ -133,4 +139,69 @@ class TabsPerformanceTest: BaseTestCase {
             counter += 1
         }
     }*/
+
+    func testPerfHistory100startUp() {
+        waitForTabsButton()
+        app.terminate()
+        measure(metrics: [
+            XCTMemoryMetric(),
+            XCTClockMetric(), // to measure timeClock Mon
+            XCTCPUMetric(), // to measure cpu cycles
+            XCTStorageMetric(), // to measure storage consuming
+            XCTMemoryMetric()]) {
+            // activity measurement here
+            app.launch()
+            waitForTabsButton()
+        }
+    }
+
+    func testPerfHistory100openMenu() {
+        waitForTabsButton()
+        measure(metrics: [
+            XCTMemoryMetric(),
+            XCTClockMetric(), // to measure timeClock Mon
+            XCTCPUMetric(), // to measure cpu cycles
+            XCTStorageMetric(), // to measure storage consuming
+            XCTMemoryMetric()]) {
+                navigator.goto(LibraryPanel_History)
+                let loaded = NSPredicate(format: "count == 101")
+                expectation(for: loaded, evaluatedWith: app.tables[AccessibilityIdentifiers.LibraryPanels.HistoryPanel.tableView].cells, handler: nil)
+                waitForExpectations(timeout: TIMEOUT, handler: nil)
+        }
+    }
+
+    func testPerfBookmarks1000startUp() {
+        waitForTabsButton()
+        app.terminate()
+        measure(metrics: [
+            XCTMemoryMetric(),
+            XCTClockMetric(), // to measure timeClock Mon
+            XCTCPUMetric(), // to measure cpu cycles
+            XCTStorageMetric(), // to measure storage consuming
+            XCTMemoryMetric()]) {
+            // activity measurement here
+            app.launch()
+            waitForTabsButton()
+        }
+    }
+
+    func testPerfBookmarks1000openMenu() {
+        waitForTabsButton()
+        measure(metrics: [
+            XCTMemoryMetric(),
+            XCTClockMetric(), // to measure timeClock Mon
+            XCTCPUMetric(), // to measure cpu cycles
+            XCTStorageMetric(), // to measure storage consuming
+            XCTMemoryMetric()]) {
+            // activity measurement here
+                navigator.goto(LibraryPanel_Bookmarks)
+                waitForExistence(app.tables["Bookmarks List"], timeout: TIMEOUT_LONG)
+                let number_cells = app.tables["Bookmarks List"].cells.count
+                // debug line - remove
+                print("Number cells \(number_cells)")
```

Comment:
Let's remove this line and the previous one as those were just for testing purposes

Reply:
done